docs/man/rpm-common.8.scd

```diff
@@ -0,0 +1,229 @@
+RPM-COMMON(8)
+
+# NAME
+rpm-common - operations and options common to the main *rpm*(8) executables
+
+# SYNOPSIS
+_RPMCMD_ [options] {*-?*|*--help*}
```

Comment:
We have **rpmcmd** in the template. Technically, this is a user-supplied argument (argument 0) so perhaps it's correct but not sure...

Reply:
Yup, I'm not sure either. `*rpmcmd* seems wrong because that's supposed to be a literal thing.

Reply:
Oh right. `*rpmcmd*` still somehow makes sense in the template, but not as much here, indeed.

In that case, `_RPMCMD_` is fine but maybe treat it like an argument, i.e. what you suggested in the below thread.